spark/src/main/java/org/apache/iceberg/spark/source/SparkBatchWrite.java

```diff
@@ -493,6 +494,9 @@ protected void setCurrentKey(PartitionKey currentKey) {
     public void write(InternalRow row) throws IOException {
       writeInternal(row);
     }
+
+    @Override
+    public void close() {}
```

Comment:
Can you add a comment to explain why this is empty? If I understand correctly, close is called after `abort` or `commit` is called. Both `abort` and `commit` will close the current data file so there is no more cleanup to do here.

Reply:
I will add this comment.